public_html/wp-content/mu-plugins/blocks/assets/src/sessions/edit.js

```diff
@@ -18,48 +18,11 @@ import SessionsBlockControls from './block-controls';
 import SessionsInspectorControls from './inspector-controls';
 import GridToolbar from '../shared/grid-layout/toolbar';
 import { ICON }                  from './index';
+import { WC_BLOCKS_STORE } from '../blocks-store';
 
 const blockData = window.WordCampBlocks.sessions || {};
-const MAX_POSTS = 100;
-
-const ALL_POSTS_QUERY = {
-	orderby  : 'title',
-	order    : 'asc',
-	per_page : MAX_POSTS,
-	_embed   : true,
-};
-
-const ALL_TERMS_QUERY = {
-	orderby  : 'name',
-	order    : 'asc',
-	per_page : MAX_POSTS,
-};
 
 class SessionsEdit extends Component {
-	constructor( props ) {
-		super( props );
-
-		this.fetchSessionDetails();
-	}
-
-	fetchSessionDetails() {
-		const allSessionPosts = apiFetch( {
-			path: addQueryArgs( `/wp/v2/sessions`, ALL_POSTS_QUERY ),
```

Comment:
Looks like we can remove the `apiFetch` and `addQueryArgs` imports here, and in other files.

Reply:
Fixed in cdd1546012afa86d46f76e3e56f91156312bf7c2

(And other commits for the other blocks)